src/clj/game/cards/events.clj

```diff
@@ -836,7 +847,8 @@
    {:msg "draw 5 cards" :effect (effect (draw 5))}
 
    "Queens Gambit"
-   {:choices ["0", "1", "2", "3"] :prompt "How many advancement tokens?"
+   {:implementation "Prevent access is not implemented"
```

Comment:
Access prevention of the targeted card _does_ work now, actually--so this can be stricken.